snippets/csharp/System.Security.Cryptography/Aes/Overview/program.cs

```diff
@@ -1,119 +1,83 @@
 {
-    class AesExample
+    public static void Main()
     {
-        public static void Main()
-        {
-            string original = "Here is some data to encrypt!";
-
-            // Create a new instance of the Aes
-            // class.  This generates a new key and initialization
-            // vector (IV).
-            using (Aes myAes = Aes.Create())
-            {
-
-                // Encrypt the string to an array of bytes.
-                byte[] encrypted = EncryptStringToBytes_Aes(original, myAes.Key, myAes.IV);
-
-                // Decrypt the bytes to a string.
-                string roundtrip = DecryptStringFromBytes_Aes(encrypted, myAes.Key, myAes.IV);
-
-                //Display the original data and the decrypted data.
-                Console.WriteLine("Original:   {0}", original);
-                Console.WriteLine("Round Trip: {0}", roundtrip);
-            }
-        }
-        //<Snippet2>
-        static byte[] EncryptStringToBytes_Aes(string plainText, byte[] Key, byte[] IV)
-        {
-            // Check arguments.
-            if (plainText == null || plainText.Length <= 0)
-                throw new ArgumentNullException("plainText");
-            if (Key == null || Key.Length <= 0)
-                throw new ArgumentNullException("Key");
-            if (IV == null || IV.Length <= 0)
-                throw new ArgumentNullException("IV");
-            byte[] encrypted;
-
-            // Create an Aes object
-            // with the specified key and IV.
-            using (Aes aesAlg = Aes.Create())
-            {
-                aesAlg.Key = Key;
-                aesAlg.IV = IV;
-
-                // Create an encryptor to perform the stream transform.
-                ICryptoTransform encryptor = aesAlg.CreateEncryptor(aesAlg.Key, aesAlg.IV);
-
-                // Create the streams used for encryption.
-                using (MemoryStream msEncrypt = new MemoryStream())
-                {
-                    using (CryptoStream csEncrypt = new CryptoStream(msEncrypt, encryptor, CryptoStreamMode.Write))
-                    {
-                        using (StreamWriter swEncrypt = new StreamWriter(csEncrypt))
-                        {
-                            //Write all data to the stream.
-                            swEncrypt.Write(plainText);
-                        }
-                        encrypted = msEncrypt.ToArray();
-                    }
-                }
-            }
+        var original = "Here is some data to encrypt!";
 
-            // Return the encrypted bytes from the memory stream.
-            return encrypted;
-        }
-        //</Snippet2>
+        // Create a new instance of the Aes
+        // class.  This generates a new key and initialization
+        // vector (IV).
+        using var myAes = Aes.Create();
+        // Encrypt the string to an array of bytes.
+        var encrypted = EncryptStringToBytes(original, myAes.Key, myAes.IV);
```

Comment:
```suggestion
        using Aes myAes = Aes.Create();
        // Encrypt the string to an array of bytes.
        byte[] encrypted = EncryptStringToBytes(original, myAes.Key, myAes.IV);
```

These uses of `var` would violate the style guide (`var` is only permissable with an as-cast, hard-cast, or `new`).